README.md

```diff
@@ -413,8 +414,9 @@ Sirius developers would like to thank:
 [Pandore]: https://clouard.users.greyc.fr/Pandore "Pandore: Une bibliothèque d'opérateurs de traitement d'images (Version 6.6). Laboratoire GREYC."
 [Getreuer]: https://doi.org/10.5201/ipol.2011.g_lmii "Getreuer, P. (2011). Linear Methods for Image Interpolation. Image Processing On Line, 1, 238259."
 
+[Doxygen]: https://CS-SI.github.io/SIRIUS/doxy_html/index.html
 [Internals]: INTERNALS.md "Internals"
-[Theoretical Basis]: https://CS-SI.github.io/SIRIUS/html/Sirius.html
+[Sirius doc]: https://CS-SI.github.io/SIRIUS/html/Sirius.html
```

Comment:
There are other links which are using `[Theoretical Basis]` reference in `README.md`:
* last line of [filter options](https://github.com/dumasl/SIRIUS/blob/76ce640ead1d5e5879a95cab00f976b1f111e36a/README.md#filter-options)
* last line of [resampling options](https://github.com/dumasl/SIRIUS/blob/76ce640ead1d5e5879a95cab00f976b1f111e36a/README.md#resampling-options)